include/EASTL/iterator.h

```diff
@@ -639,6 +639,9 @@ namespace eastl
 		back_insert_iterator& operator=(const_reference value)
 			{ container.push_back(value); return *this; }
 
+		back_insert_iterator& operator=(typename Container::value_type && value)
```

Comment:
Please align the move-semantic reference with value_type.

Reply:
Do you mean something like
>back_insert_iterator& operator=(typename Container::value_type&& value)
?

Reply:
Yes please.